tests/fixtures/apps/demultiplexing/flow_cells/160202_ST-E00266_0064_AHKHHGCCXX/SampleSheet.csv

```diff
@@ -0,0 +1,10 @@
+[Data]
+FCID,Lane,SampleID,SampleRef,index,SampleName,Control,Recipe,Operator,Project
+HKHHGCCXX,1,ADM1286A18_dual11,hg19,TCCGGAGA,<anonymous_1>,N,R1,NN,1
+HKHHGCCXX,2,ADM1286A19_dual19,hg19,CGCTCATT,<anonymous_1>,N,R1,NN,1
+HKHHGCCXX,3,ADM1286A20_dual27,hg19,GAGATTCC,<anonymous_1>,N,R1,NN,1
+HKHHGCCXX,4,ADM1237A16_dual44,hg19,GAATTCGT,<anonymous_2>,N,R1,NN,2
+HKHHGCCXX,5,ADM1280A1_dual72,hg19,CGGCTATG,<anonymous_3>,N,R1,NN,3
+HKHHGCCXX,6,ADM1282A1_dual80,hg19,TCCGCGAA,<anonymous_4>,N,R1,NN,4
+HKHHGCCXX,7,ADM1283A1_dual88,hg19,TCTCGCGC,<anonymous_5>,N,R1,NN,5
+HKHHGCCXX,8,ADM1284A1_dual96,hg19,AGCGATAG,<anonymous_6>,N,R1,NN,6

```

Comment:
Do we want to remove the sample internal_id's here?

Reply:
fixed